.env_example

```diff
@@ -23,13 +23,5 @@ AE_NETWORK_ID_PROD="ae_devnet"
 AE_BACKEND="fate"
 AE_ABI_VERSION=3
 
-BACKENDLESS_LOGIN="xxxxxx"
-BACKENDLESS_PASSWORD="xxxxxxx."
-BACKENDLESS_ID="xxxxxx"
-BACKENDLESS_KEY="xxxxx"
-BACKENDLESS_URL="xxxxxxx"
-BACKENDLESS_USERTOKEN="xxxxx"
-BACKENDLESS_TABLE="xxxxx"
-
 BEARER_AUTH_TOKEN="Bearer xxxxx"
 BEARER_AUTH_TOKEN_PROD="Bearer xxxxxx"
```

Comment:
do we need th prod_token in the examples? Prod env vars should be available only for production

Reply:
those env vars are removed in 'added docs' PR